Standard.json

```diff
@@ -25,6 +25,13 @@
                 "description": "The type of the VM to deploy. Note that only NV-series support gaming."
             }
         },
+        "isSpotVM": {
+            "type": "bool",
+            "metadata": {
+                "description": "Whether the VM is a Spot VM or not. Spot VMs let you use unused Azure resources for a cheaper price but can get force stopped at anytime."
```

Comment:
Recommend adding a note saying that spot does not work with promo skus.

Reply:
Done !